web/src/auth/LoginPage.js

```diff
@@ -383,6 +383,28 @@ class LoginPage extends React.Component {
             }
           };
 
+          const changePasswordForm = () => {
```

Comment:
Don't add more code to `LoginPage.js`, it's already too complicated

Reply:
@hsluoyz should I move new code from login.js as much as possible but keep 1-2 new lines? 
Or don't modify login.is at all?

Reply:
1-2 new lines are OK

Reply:
I've moved wrap code to ChangePasswordForm, 
please take a look, it's still not 1-2 line but esentialy only widget declaration and line that call this widget, 
I think possible to move more to dedicated function but not sure if it will be more readable and predictable.
Can try if still to much for LoginPage.js

Reply:
OK